src/wasm-type.h

```diff
@@ -316,6 +317,7 @@ struct Struct {
   Struct(FieldList&& fields) : fields(std::move(fields)) {}
   bool operator==(const Struct& other) const { return fields == other.fields; }
   bool operator!=(const Struct& other) const { return !(*this == other); }
+  bool operator<(const Struct& other) const { return fields < other.fields; }
```

Comment:
I wish we could use C++20's spaceship operator here instead of having to define both `operator==` and `operator<` :(